mx.c

```diff
@@ -1685,6 +1685,92 @@ struct Mailbox *mx_path_resolve(const char *path)
   return m;
 }
 
+/**
+ * mx_mbox_find_by_name - Find a Mailbox with given name under an account.
+ * @param a    Account to search
+ * @param name Name to find
+ * @retval ptr Mailbox
+ */
+static struct Mailbox *mx_mbox_find_by_name_ac(struct Account *a, const char *name)
+{
+  if (!a || !name)
+    return NULL;
+
+  struct MailboxNode *np = NULL;
+
+  STAILQ_FOREACH(np, &a->mailboxes, entries)
+  {
+    if (np->mailbox->name && mutt_str_equal(np->mailbox->name, name))
+      return np->mailbox;
+  }
+
+  if (!np)
+    return NULL;
+  return np->mailbox;
+}
+
+/**
+ * mx_mbox_find_by_name - Find a Mailbox with given name.
+ * @param name Name to search
+ * @retval ptr Mailbox
+ */
+static struct Mailbox *mx_mbox_find_by_name(const char *name)
+{
+  if (!name)
+    return NULL;
+
+  char buf[PATH_MAX];
+  mutt_str_copy(buf, name, sizeof(buf));
+
+  struct Account *np = NULL;
+  TAILQ_FOREACH(np, &NeoMutt->accounts, entries)
+  {
+    struct Mailbox *m = mx_mbox_find_by_name_ac(np, buf);
+    if (m)
+      return m;
+  }
+
+  return NULL;
+}
+
+/**
+ * mx_path_resolve - Find a Mailbox with a given name
+ * @param name  Mailbox name
+ * @retval ptr  Mailbox
+ */
+static struct Mailbox *mx_name_resolve(const char *name)
```

Comment:
well now this is a simple wrapper around `mx_mbox_find_by_name`. Do you still want to keep it?

Reply:
Good point, thanks